cli/tools/test/mod.rs

```diff
@@ -393,6 +405,34 @@ impl TestFailure {
     }
   }
 
+  pub fn error_location(&self) -> Option<TestLocation> {
+    match self {
+      // TODO: What happens if not enough call stack frames were recorded?
+      // TODO: Could anything else trigger the ext:cli/40_test.js file? Maybe we should also test for the method name
+      TestFailure::JsError(js_error)
+      | TestFailure::Expected { js_error, .. } => js_error
+        .frames
+        .iter()
+        .position(|v| v.file_name.as_deref() == Some("ext:cli/40_test.js"))
+        // Go one up in the stack frame, this is where the user code was
+        .and_then(|index| index.checked_sub(1))
+        .and_then(|index| {
+          let user_frame = &js_error.frames[index];
+          let file_name = user_frame.file_name.as_ref()?.to_string();
+          // Turn into zero based indices
+          let line_number = user_frame.line_number.map(|v| v - 1)? as u32;
+          let column_number =
+            user_frame.column_number.map(|v| v - 1).unwrap_or(0) as u32;
+          Some(TestLocation {
```

Comment:
This info lets us show an indicator at the `assertEquals()` that actually failed.

I couldn't really find a better way of figuring out where in the user code the error happened. Is this really it?